tags/all/families.csv

```diff
@@ -4105,61 +3930,71 @@ Lemon,/Theme/Brush,100
 Lemonada,/Expressive/Calm,55
 Lemonada,/Expressive/Playful,12
 Lemonada,/Theme/Brush,100
-Lexend Deca,/Expressive/Calm,100
+Lexend Deca,/Expressive/Calm,99.8
 Lexend Deca,/Sans/Geometric,100
-Lexend Exa,/Expressive/Calm,100
+Lexend Exa,/Expressive/Calm,99.8
 Lexend Exa,/Expressive/Competent,95.9
 Lexend Exa,/Sans/Geometric,100
-Lexend Giga,/Expressive/Calm,100
+Lexend Giga,/Expressive/Calm,99.8
 Lexend Giga,/Expressive/Competent,95.9
 Lexend Giga,/Sans/Geometric,100
-Lexend Mega,/Expressive/Calm,100
+Lexend Mega,/Expressive/Calm,99.8
 Lexend Mega,/Expressive/Competent,95.9
 Lexend Mega,/Sans/Geometric,100
-Lexend Peta,/Expressive/Calm,100
+Lexend Peta,/Expressive/Calm,99.8
 Lexend Peta,/Expressive/Competent,95.9
 Lexend Peta,/Sans/Geometric,100
-Lexend Tera,/Expressive/Calm,100
+Lexend Tera,/Expressive/Calm,99.8
 Lexend Tera,/Expressive/Competent,95.9
 Lexend Tera,/Sans/Geometric,100
-Lexend Zetta,/Expressive/Calm,100
+Lexend Zetta,/Expressive/Calm,99.8
 Lexend Zetta,/Expressive/Competent,95.9
 Lexend Zetta,/Sans/Geometric,100
-Lexend,/Expressive/Calm,100
+Lexend,/Expressive/Calm,99.8
 Lexend,/Sans/Geometric,100
+Libre Barcode 128 Text,/Expressive/Futuristic,99

```

Comment:
I see these have got reintroduced. I'll take them out tomorrow

Reply:
Evan needs this merged. Marking this as resolved to unblock the merge.